core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -110,20 +108,30 @@ object WorkspaceService {
       terraBucketWriterRole,
       rawlsWorkspaceAclManager,
       multiCloudWorkspaceAclManager,
-      fastPassServiceConstructor,
+      (context: RawlsRequestContext) => fastPassServiceConstructor(context, dataSource),
       new WorkspaceRepository(dataSource),
       new BillingRepository(dataSource)
     )
 
-  val SECURITY_LABEL_KEY = "security"
-  val HIGH_SECURITY_LABEL = "high"
-  val LOW_SECURITY_LABEL = "low"
+  val SECURITY_LABEL_KEY: String = "security"
+  val HIGH_SECURITY_LABEL: String = "high"
+  val LOW_SECURITY_LABEL: String = "low"
+  val BUCKET_GET_PERMISSION: String = "storage.buckets.get"
 
-  val BUCKET_GET_PERMISSION = "storage.buckets.get"
+  case class QueryOptions(options: Set[LookupExpression], attrSpecs: WorkspaceAttributeSpecs) {
```

Comment:
This was just to simplify the parameters for `getWorkspace` and `getWorkspaceById`.